doc/plugins_manual/introduction.tex

```diff
@@ -0,0 +1,46 @@
+\section{Introduction}
+
+RAVEN (Risk Analysis Virtual ENvironment) [TODO cite] is a powerful tool for risk- and uncertainty-based stochastic analysis. TODO say more.
```

Comment:
can you add the citation and comment out the "TODO say more"?

Reply:
done